spec/services/api/v2/sections/chapters_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+RSpec.describe Api::V2::Sections::ChaptersController do
+
+  # '/api/v2/sections/:id/chapters'
+  describe '#index', type: :controller do
+    let(:params) { { section_id: section.id } }
+    let(:section) { create(:chapter, :with_section).section }
+
+    subject(:do_request) do
+      get(:index, params:, format: 'csv')
+
+      response
+    end
+
+    it { is_expected.to be_successful }
+
+    it { is_expected.to have_attributes media_type: /csv/ }
+
+    context "when the param 'as_of' is specified" do
```

Comment:
It actually does this behaviour without the as_of

Reply:
Also, I think we can move this to the shared examples, actually. Since this is actually part of the AC of all of the tickets :tada: